style.css

```diff
@@ -0,0 +1,167 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
+
+body {
+  display: flex;
+  flex-direction: column;
+  background-color: #f8f9fa;
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  overflow-x: hidden;
+}
+
+.header {
+  display: flex;
+  width: 375px;
+  justify-content: center;
+  justify-content: space-between;
+  align-items: center;
+  padding: 50px 36px 15px 24px;
+  box-sizing: border-box;
+  background-color: #fff;
+  margin: 0 auto;
+}
+
+#logo {
+  font-family: "Poppins", sans-serif;
+  width: 76px;
+  height: 20px;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+#menu-nav-mobile {
+  background-color: transparent;
+  transition: all 0.3s ease-in-out;
+}
+
+#menu-nav-mobile.active {
+  display: block;
```

Comment:
Great job @Sonickmumba 👏

- [ ] [Optional] Try to use class selectors instead of IDs to select multiple properties